scipy/linalg/_sketches.py

```diff
@@ -45,10 +45,10 @@ def clarkson_woodruff_transform(input_matrix, sketch_size):
     """
     Given an input_matrix  of size (n, d), compute a matrix A' of size  (sketch_size, d)
     which holds:
-    $||Ax|| = (1 \pm \epsilon) ||A'x||$
```

Comment:
Note that math characters are fine, but you need to wrap them in a `math:` directive. See e.g. https://github.com/scipy/scipy/blob/master/scipy/special/_spherical_bessel.py#L7 for an example. Note that you will also need to prefix the string with an `r` to prevent an overabundance of slashes.

Reply:
Hi @person142 ,

Thanks for the info. I just rewrote it. Let's go to see if it passes all the CI tasks now.